src/main/java/com/forest/mytopmovies/config/security/NoRedirectStrategy.java

```diff
@@ -0,0 +1,13 @@
+package com.forest.mytopmovies.config.security;
```

Comment:
I deleted this and authenticated without redirect so I'm not sure what problem this is solving

Reply:
Have you had a chance to investigate this yet?

Reply:
No I will leave this one to next sprint. It's on my diary.